experimenter/tests/integration/nimbus/test_create_mobile_experiment.py

```diff
@@ -5,21 +5,25 @@
 
 import pytest
 import requests
-from nimbus.models.base_dataclass import BaseExperimentApplications
+
 from nimbus.pages.experimenter.summary import SummaryPage
 from nimbus.utils import helpers
 
 
 @pytest.fixture
-def experiment_slug():
-    return "firefox-ios-integration-test"
+def experiment_slug(application):
+    return f"firefox-{application.lower()}-integration-test-experiment"
 
 
-@pytest.fixture
-def default_data_api(default_data_api):
-    feature_config_id = helpers.get_feature_id_as_string(
-        "messaging", BaseExperimentApplications.FIREFOX_IOS.value
-    )
+def test_create_mobile_experiment_for_integration_test(
+    selenium, experiment_url, kinto_client, default_data_api, experiment_slug, application
+):
+    """Create a mobile experiment for device integration tests"""
+    apps = ["IOS", "FENIX"]
+    logging.info(application)
+    if str(application) not in apps:
+        pytest.skip()
```

Comment:
Is this just for debugging?

Reply:
No, we don't need to create recipes for anything other than ios, or fenix. So this just skips the test if the `application` is neither one of them.

Reply:
Oh like, require the parameterized application fixture, which creates an instance for every application, but skip all but those two.  I had to deal with this recently, there was a way to do it with just the decorators.  I can't remember.  Anyways that makes sense.

Reply:
Yeah I could do it with decorators